API-Samples/android/project_template/build.gradle

```diff
@@ -32,13 +32,13 @@ if(!ndkDir || ndkDir.empty) {
 def stlType = 'c++_static'
 
 android {
-    compileSdkVersion  24
-    buildToolsVersion '25.0.0'
+    compileSdkVersion  26
+    buildToolsVersion '28.0.2'
```

Comment:
delete this line: let Android studio find the matching tools version for "compileSdkVersion"